### DISCUSSION TITLE
Welcome to ChessEngine Discussions!

### DISCUSSION THREAD

**Discussion by MrMystery10-del** [MrMystery10-del/ChessEngine#4] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vansh2552** [comment]
Hello, I want to contribute to this project.
can you please brief me how can I set this repo locally and start contributing. I haven't contributed to before so it will be a great help if you could help me out here. I'm really interested in contributing.
Thanks

**Reply by MrMystery10-del** [reply]
To facilitate the setup process due to its structure, it is recommended that you use Intellij IDEA as your IDE. If you choose a different IDE, you will have to manually configure the libraries located in the lib folder and link them to your project. Additionally, ensure that you are using JDK 17 as the Java version. Once the setup is complete, you can run the Core file and launch the application.

**Reply by vansh2552** [reply]
Thanks for your reply. I tried running the code but I am facing some errors, I think is because of the configurations. Can you let me know what configuration should I choose in the configuration template? 
![image](https://user-images.githubusercontent.com/87716952/233637339-02858f0f-1916-4b5a-83eb-7b6979d612fc.png)

**Reply by vansh2552** [reply]
Never mind, the code is running now. I will start contributing now.
